addons/io_hubs_addon/components/ui.py

```diff
@@ -229,7 +516,18 @@ def unregister():
     bpy.utils.unregister_class(HubsMaterialPanel)
     bpy.utils.unregister_class(HubsBonePanel)
     bpy.utils.unregister_class(TooltipLabel)
+    bpy.utils.unregister_class(HubsUpdateSceneOperator)
+    bpy.utils.unregister_class(HubsCreateRoomOperator)
+    bpy.utils.unregister_class(HUBS_PT_ToolsPanel)
+    bpy.utils.unregister_class(HubsSceneDebuggerRoomCreatePrefs)
+
+    del bpy.types.Scene.hubs_scene_debugger_room_create_prefs
+    del bpy.types.Scene.hubs_scene_debugger_expanded
 
     bpy.types.TOPBAR_MT_window.remove(window_menu_addition)
     bpy.types.VIEW3D_MT_object.remove(object_menu_addition)
     bpy.types.VIEW3D_PT_gizmo_display.remove(gizmo_display_popover_addition)
+
+    global web_driver
+    if web_driver and isWebdriverAlive(web_driver):
+        web_driver.close()
```

Comment:
Well, I did some more checking and it turns out I was working on outdated information.  It turns out that Blender has called unregister on exit since versions 2.91.0 and 2.83.9[1][2], and my own tests confirm this.  But, you say you think you saw a few instances where it wasn't being called, so now I'm even more curious.  Do you remember which exit situations didn't call unregister?

[1] https://projects.blender.org/blender/blender/commit/fa566157a5c351775d082b05b180c630665b4afc)
[2] https://blender.stackexchange.com/questions/153084/blender-add-on-dev-unregister-is-never-called-on-blender-exit

Reply:
I can't, I'm going to remove it from now, we can always add it it we find a good reason.